lib/judges/baza.rb

```diff
@@ -219,6 +210,66 @@ def headers
     }
   end
 
+  def request_options_for_push(data, meta)
```

Comment:
@tank-bohr I believe, it's not a good idea to make many small methods just to make code "readable". This piece of code would definitely look better if we inline it.